packages/run-protocol/src/vaultFactory/vaultDirector.js

```diff
@@ -72,13 +80,17 @@ const initState = (zcf, directorParamManager, debtMint) => {
 
   const vaultParamManagers = makeScalarMap('brand');
 
+  // XXX need a way to register this data stream for consumption
+  const { updater: econMetrics } = makeNotifierKit();
```

Comment:
@gibson042 I punted on mocking this because I wasn't sure where to put it. seems like @dtribble may have some designs in mind so I don't want to proceed too far.